pkg/smbios/info_linux_test.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package smbios
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestFromSysfs(t *testing.T) {
+	testutil.SkipIfNotRoot(t)
+
+	if runtime.GOARCH != "amd64" {
+		t.Skip("Test only supported in amd64 Qemu")
```

Comment:
How do you know you are in Qemu? Why not change the file name
info_linux_amd64_test.go